package-generator/tests/unit/test_product.py

```diff
@@ -114,3 +124,13 @@ def test_ports(self):
         product = Product(PRODUCT_NAME, PRODUCT_VERSION, metadatas)
         self.assertEquals(len(product.get_tcp_ports()), 2)
         self.assertEquals(len(product.get_udp_ports()), 1)
+
+    @mock.patch('util.utils_clients.util_apis')
+    def test_obtain_images(self, mock_image):
+        """test the functionalities to get product images"""
+        mock_image._get_image_name.return_value = "nameID"
+        Config.Clients = mock_image
```

Comment:
Maybe I do not see anything in the code But you are assiging the mock_image to the Config.Clients but you are using the Product class. I do not see the relationship between Config and Product.